### DISCUSSION TITLE
How to install?

### DISCUSSION THREAD

**Discussion by Dani94SCC** [ivangabriele/clamav-desktop#61] [Questions & Answers]
Sorry, I have downloaded the zip file, but how to install? I didn't find any wiki. Sorry for the noob question.

Fedora36

**Comment by ivangabriele** [comment]
Hi @Dani94SCC,

Sorry for the late answer. I'm working on the v0.4 version which is rewritten in Rust. Once it's released, I'll add a complete documentation on how to install it ^^.

**Comment by maj113** [comment]
> Sorry for the late answer. I'm working on the v0.4 version which is rewritten in Rust. Once it's released, I'll add a complete documentation on how to install it ^^.

Is this being worked on?

**Comment by ivangabriele** [comment]
@maj113 It is: https://github.com/ivangabriele/clamav-desktop/pull/41, the rewriting takes a bit of time and I only work on this project during my free time.

**Comment by beedell-roke** [comment]
~~This should probably be converted into a Q&A Discussion rather than an issue.~~
